src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -221,6 +224,8 @@ public MockResponse dispatch(RecordedRequest request) throws InterruptedExceptio
                             .build(), "sa.stopped");
                     return new MockResponse().setResponseCode(200)
                             .addHeader("Content-Type", "application/json; charset=utf-8");
+                } else if (path.matches("/v1/directories/.*")) {
```

Comment:
it's better if you put the exact route. Here any calls to directory server will return 200 so you don't test when the call fail. And it'll be easier for anyone to add new tests without having to touch your tests.